src/test/java/one/util/streamex/MoreCollectorsTest.java

```diff
@@ -591,6 +595,242 @@ public void testToEnumSet() {
         checkCollectorEmpty("Empty", EnumSet.noneOf(TimeUnit.class), MoreCollectors.toEnumSet(TimeUnit.class));
     }
 
+    /**
+     * See {@link MoreCollectors#entriesToMap()}.
+     */
+    @Test
+    public void testEntriesToMap() {
+        assertThrows(IllegalStateException.class, () ->
+                EntryStream.generate(() -> "a", () -> 1).limit(10).collect(MoreCollectors.entriesToMap()));
```

Comment:
Better to assert the exception message text as well. I believe we don't have a helper for this. You may write a helper or just implement it manually.